quora-service/src/main/java/com/upgrad/quora/service/dao/QuestionDao.java

```diff
@@ -1,5 +1,6 @@
 package com.upgrad.quora.service.dao;
 
+import com.upgrad.quora.service.entity.AnswerEntity;
```

Comment:
remove this line, if answer entity is not used

Reply:
Made changes as per code review